### DISCUSSION TITLE
First time installing and configuring ZOAU

### DISCUSSION THREAD

**Discussion by gngrossi** [IBM/zoau-samples#52] [Q&A] [answered]
In the ZOAU documentation, there is a step for enabling logging (gather debugging information) by creating a logging_config.py file for Python programs.  https://www.ibm.com/docs/en/zoau/1.2.x?topic=installing-configuring-zoau

Is this step done as a default for all .py programs and if so, where would/does it reside?

Thanks,
Gary

**Comment by EnsonoStevenPerva** [comment, marked as answer]
You could create this if you need the additional logging. 

The logging_config.py program would reside in some directory (often the same as your Python program) and you would import it using one of the typical Python import mechanisms. There's a bunch of different ways to do this, but the simplest is to have the logging_config.py file in the same directory as your program and simply do an `import logging_config` as you see in the documentation. 

Hope this helps!
